### PR TITLE
Change `UserUpdateByAdmin` to be Django class based view 

### PR DESCRIPTION
**Pull request by dybi**
resolves: https://github.com/Code-Poets/sheetstorm/issues/248

### CODE REVIEW

File: users/tests/test_views.py [hunk 1]
```diff
@@ -156,3 +156,56 @@ def test_user_update_view_should_not_update_user_on_post_if_form_is_invalid(self
         self.user.refresh_from_db()
         self.assertEqual(response.status_code, 200)
         self.assertEqual(phone_number_before_request, self.user.phone_number)
+
+
+class UserUpdateByAdminTests(TestCase):
+    def setUp(self):
+        self.user = CustomUser(
```
Comment (rwrzesien) [inline users/tests/test_views.py]: Please use factory for this.

File: users/templates/users_detail.html [hunk 2]
```diff
@@ -32,8 +32,8 @@ <h1>
 
   </div>
     <form class="form-horizonal" action="{% url 'custom-user-update-by-admin' pk=user_detail.pk %}" method="POST">
-      {% csrf_token %}
-      {% render_form serializer template_pack='rest_framework/vertical' %}
+        {% csrf_token %}
```
Comment (rwrzesien) [inline users/templates/users_detail.html]: 2-spaces "tab" is ok in HTML, please align.

File: users/tests/test_views.py [hunk 3]
```diff
@@ -156,3 +156,56 @@ def test_user_update_view_should_not_update_user_on_post_if_form_is_invalid(self
...
+        self.user_admin = AdminUserFactory()
+        self.client.force_login(self.user_admin)
+        self.user.refresh_from_db()
+
+    def test_user_update_by_admin_view_should_display_user_details_on_get(self):
+        response = self.client.get(path=reverse("custom-user-update-by-admin", args=(self.user.pk,)))
+        self.assertEqual(response.status_code, 200)
+        self.assertContains(response, self.user.email)
+        self.assertContains(response, self.user.first_name)
+        self.assertContains(response, self.user.last_name)
```
Comment (rwrzesien) [inline users/tests/test_views.py]: You can also add check like `self.assertTemplateUsed('template_name.html')`

File: users/tests/test_views.py [hunk 4]
```diff
@@ -156,3 +156,56 @@ def test_user_update_view_should_not_update_user_on_post_if_form_is_invalid(self
...
+        self.user.refresh_from_db()
+        self.assertEqual(response.status_code, 302)
+        self.assertEqual(new_phone_number, self.user.phone_number)
+
+    def test_user_update_by_admin_view_should_not_update_user_on_post_if_form_is_invalid(self):
+        phone_number_before_request = self.user.phone_number
+        invalid_phone_number = generate_random_phone_number(constants.PHONE_NUMBER_MIN_LENGTH - 1)
+        response = self.client.post(
+            path=reverse("custom-user-update-by-admin", args=(self.user.pk,)),
+            data={
```
Comment (rwrzesien) [inline users/tests/test_views.py]: Looks like this `data` could be extracted and defined in `setUp` ie.

File: users/templates/users_detail.html (hunk 2 above)
Comment (dybi) [reply users/templates/users_detail.html]: :+1:

File: users/tests/test_views.py (hunk 3 above)
Comment (dybi) [reply users/tests/test_views.py]: :+1:

File: users/tests/test_views.py (hunk 4 above)
Comment (dybi) [reply users/tests/test_views.py]: `data` is not exactly the same, so it cannot be a field, but I've made a small helper method for it :)

**Pull request merged**